### PR TITLE
Adjust some idea plugin behaviors

### PR DESCRIPTION
**Pull request by takahirom**
* Roborazzi Plugin will use default configuration of IDE
* Sort Gradle tasks and select first by default

https://github.com/user-attachments/assets/76559bf1-3d90-4a93-9bb5-ab708e01de17

### CODE REVIEW

**Comment by takahirom** [comment]
@eyedol Thank you for all your contributions to the Roborazzi Idea Plugin! I've made some adjustments to the behaviors related to executing Gradle tasks. Could you please review these changes?
[https://github.com/takahirom/roborazzi/pull/431](https://github.com/takahirom/roborazzi/pull/431%E2%80%8B)

File: roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/PreviewViewModel.kt [hunk 1]
```diff
@@ -256,7 +251,18 @@ class PreviewViewModel {
...
+  }
+
+  fun onTaskExecuted(project: Project) {
+    _dropDownUiState.update { currentUiState ->
+      currentUiState.copy(flag = ActionToolbarUiState.Flag.IDLE)
+    }
+    refreshImageList(project)
+    // For updating the task list after syncing the gradle task
+    coroutineScope.launch {
+      fetchTasks(project)
```
Comment (eyedol) [inline roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/PreviewViewModel.kt:264]: I guess we can skip executing this in a coroutine

File: roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/RoborazziGradleTask.kt [hunk 2]
```diff
@@ -24,46 +26,53 @@ class RoborazziGradleTask {
...
+    val runManager = RunManager.getInstance(project)
+    val configurationFactory = GradleExternalTaskConfigurationType.getInstance().factory
+    val runConfiguration = runManager.createConfiguration(
+      "Execute $taskName",
+      configurationFactory
+    )
+
+    val gradleRunConfiguration = runConfiguration.configuration as GradleRunConfiguration
+    gradleRunConfiguration.settings.apply {
+      externalProjectPath = project.basePath
```
Comment (eyedol) [inline roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/RoborazziGradleTask.kt:52]: Neat to see it added to run configuration

**Review by eyedol — APPROVED** [review]
LGTM. Nice too see the support with running the selected tasks via the user's run configuration

File: roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/PreviewViewModel.kt (hunk 1 above)
Comment (takahirom) [reply roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/PreviewViewModel.kt:264]: Thanks
I addressed some performance issues in https://github.com/takahirom/roborazzi/pull/439/files

**Pull request merged**